whylog/tests/tests_config/test_propose_parser_name.py

```diff
@@ -0,0 +1,42 @@
+import os.path
+from unittest import TestCase
+
+from ordered_set import OrderedSet
+
+from whylog.config import YamlConfig
+
+path_test_files = ['whylog', 'tests', 'tests_config', 'test_files']
+
+
+class TestBasic(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        path = os.path.join(*path_test_files)
+        parsers_path = os.path.join(path, 'parsers.yaml')
+        rules_path = os.path.join(path, 'rules.yaml')
+        log_type_path = os.path.join(path, 'log_types.yaml')
+
+        cls.sample_line1 = "2016-04-12 23:54:45 Connection error occurred on comp1. Host name: host1"
+        cls.regex1 = "^(\d\d\d\d-\d\d-\d\d \d\d:\d\d:\d\d) Connection error occurred on (.*)\. Host name: (.*)$"
+
+        cls.config = YamlConfig(parsers_path, rules_path, log_type_path)
+
+    def test_is_free_parser_name(self):
```

Comment:
Maybe add a comment saying that following names are loaded from `parsers.yaml`. I did't know that. I had to think a while why they aren't free.